Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.h

```diff
@@ -153,7 +153,7 @@ class MediaPlayerPrivateGStreamerBase : public MediaPlayerPrivateInterface
     void attemptToDecryptWithInstance(const CDMInstance&) override;
 #endif
 #if USE(OPENCDM)
-    bool isInitDataCached(const Ref<SharedBuffer>&);
+    bool findAndSetPendingProtectionEventByInitData(const Vector<uint8_t>&, const uint32_t);
```

Comment:
can drop the const for the int, it's needless for value types

Reply:
Fixed